.env.proton

```diff
@@ -45,11 +45,21 @@ HAPI_COINGECKO_API_TOKEN_ID=proton
 HAPI_REWARDS_TOKEN=XPR
 HAPI_SYNC_STATS_INTERVAL=60
 HAPI_SYNC_SCHEDULE_HISTORY_INTERVAL=86400
+HAPI_EOS_MECHANICS_INCLUDE_TRANSACTION=
+HAPI_EOS_FAUCET_ACCOUNT=
+HAPI_EOS_FAUCET_ACCOUNT_PASSWORD=
+HAPI_EOS_STATE_HISTORY_PLUGIN_ENDPOINT=
+GOOGLE_APPLICATION_CREDENTIALS=
+HAPI_RE_CAPTCHA_PROJECT_ID=
+HAPI_PUBLIC_RE_CAPTCHA_KEY=
+HAPI_CREATE_ACCOUNT_ACTION_NAME=
+HAPI_EOS_GET_FAUCET_ACCOUNT=
+
 
 #webapp
 PORT=3000
 REACT_APP_TITLE=Proton Network Dashboard
-REACT_APP_TAG=dev
+REACT_APP_VERSION=dev
```

Comment:
We are no longer hosting this on IPFS

use this URL instead : https://eosio.online/images/proton.png